### ISSUE TITLE
Air Alarm sprites still keeping alarm light despite no local or area issues

### ISSUE THREAD

**Issue by cacogen** [tgstation/tgstation#59043]
Reporting client version: 514.1554

## Round ID:
[162039](https://scrubby.melonmesa.com/round/162039)

## Testmerges:
- [Removes bad return from processables that broke cooking ðŸ€](https://www.github.com/tgstation/tgstation/pull/58983)

## Reproduction:

This can be manually reset by changing the mode (e.g. from Draught to Contaminated).

**Comment by zxaber** [comment]
Is this related to how you have to clear the alarm? Because that's been a thing forever and I assumed it was intentional

**Comment by cacogen** [comment]
No, everything is green. There's no atmosphere alarm. A lot of them reset their sprite by themselves but some don't. I'll get a screenshot next time I encounter it.

**Comment by timothymtorres** [comment]
May be related to #11956
